tests/functional/lang/eval-fail-addErrorContext-example.flags

```diff
@@ -0,0 +1 @@
+--eval --strict --no-show-trace
```

Comment:
Missing EOL at end of file https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

Reply:
Wasn't meant to be a line, but bash handles that fine.